Add EC port requirements (#2680)
replicatedhq/replicated-docs: docs/enterprise/installing-embedded.mdx

#### Limitation

The `--pod-cidr` and `--service-cidr` flags are not supported on Red Hat Enterprise Linux (RHEL) 9 operating systems.

> ## Relocate Admin Console and LAM Ports

Author: ^ Added a new section to the Install topic

> By default, the Admin Console and Local Artifact Mirror (LAM) run on ports 30000 and 50000, respectively. If these ports are occupied, you can relocate them.

Reviewer: You'd relocate the service, not the port. So I think this sounds better.
```suggestion
By default, the Admin Console and Local Artifact Mirror (LAM) run on ports 30000 and 50000, respectively. If these ports are occupied, you can select different ports.
```

> To relocate the Admin Console and LAM ports during installation with Embedded Cluster, use the `--admin-console-port` and `--local-artifact-mirror` flags with the Embedded Cluster `install` command.

Reviewer: And then maybe something like, "To choose different ports" or whatever.

> To relocate the Admin Console and LAM ports during installation with Embedded Cluster, use the `--admin-console-port` and `--local-artifact-mirror` flags with the Embedded Cluster `install` command.

Reviewer: ```suggestion
To relocate the Admin Console and LAM ports during installation with Embedded Cluster, use the `--admin-console-port` and `--local-artifact-mirror-port` flags with the Embedded Cluster `install` command.
```

For example:

```bash
install --admin-console-port=20000 --local-artifact-mirror-port=40000
```

> The default Embedded Cluster host preflight checks verify the default ports or the user-supplied ports, so users will be alerted before installation if they need to choose new port(s). For more information about the host preflight checks for Embedded Cluster, see [About Host Preflight Checks](/vendor/embedded-overview#about-host-preflight-checks).

Reviewer: I don't even think we need to call these default. They're just the EC host preflight checks.
```suggestion
The Embedded Cluster host preflight checks verify the default ports or the user-supplied ports, so users will be alerted before installation if they need to choose new port(s). For more information about the host preflight checks for Embedded Cluster, see [About Host Preflight Checks](/vendor/embedded-overview#about-host-preflight-checks).
```

### Limitation

> It is not possible to relocate the Admin Console port during a restore with Embedded Cluster. For more information, see [Disaster Recovery for Embedded Cluster (Alpha)](/vendor/embedded-disaster-recovery).

Reviewer: same relocate wording change here
